nva-datamodel-java/src/main/java/no/unit/nva/model/associatedartifacts/file/PublisherVersion.java

```diff
@@ -2,18 +2,19 @@
 
 import static java.lang.String.format;
 import static java.util.Arrays.stream;
-import static java.util.Objects.isNull;
 import static java.util.stream.Collectors.joining;
 import com.fasterxml.jackson.annotation.JsonValue;
 import no.unit.nva.model.Revision;
+import nva.commons.core.StringUtils;
 
 public enum PublisherVersion {
     PUBLISHED_VERSION("PublishedVersion"),
+    DRAFT("draft"),
+    SUBMITTED_VERSION("SubmittedVersion"),
+    UPDATED_VERSION("UpdatedVersion"),
```

Comment:
Names copied straight from the task. Feel free to suggest other names.

Reply:
I would go for "Draft" instead of "draft", so it matches the rest of enums

Reply:
Good catch :)